src/pudl/analysis/ferc1_eia.py

```diff
@@ -1230,11 +776,11 @@ def prettyify_best_matches(
             warnings.warn(message)
 
     _log_match_coverage(connects_ferc1_eia)
-    for match_type in ["all", "overrides"]:
+    for match_set in ["all", "overrides"]:
```

Comment:
I changed `match_type` to `match_set` because `match_type` is a column in the output matches dataframe and it was confusing me. But maybe this is too tennis themed now.

Reply:
🎾 🤣  no i think this makes sense and it is helpful that it is seperate